_data/toc.yaml

```diff
@@ -1439,12 +1439,12 @@ manuals:
   - sectiontitle: Install Compose
     section:
       - path: /compose/install/
-        title: Install Docker Compose 
-      - path: /compose/install/compose-desktop/
-        title: Install Compose plugin through Desktop 
-      - path: /compose/install/compose-plugin/
-        title: Install Docker Compose Plugin 
-      - path: /compose/install/uninstall/ 
+        title: Install Docker Compose
+      - path: /compose/install/compose-linux/
+        title: Install on Linux
+      - path: /compose/install/compose-other/
```

Comment:
Could we name these pages;

- `linux` (`/compose/install/linux/`)
- `other` (`/compose/install/other/`)

It makes the URLs a bit shorter (and possibly more predictable), and prevents duplicating `compose` in the URL (we're in the compose section, so the URL already starts with `/compose/`).

Reply:
Changed per suggestion.